components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/annotations/metadata_general.html

```diff
@@ -105,6 +105,8 @@
                     $("#link_info_popup").show();
                 {% if webclient_path %}
```

Comment:
if you want to maintain the old logic then you need to keep `request.build_absolute_uri(reverse('webindex'))` rather then JS created link twice. The code below has no sense to me.